surreal.js

```diff
@@ -211,10 +245,11 @@ var $ = { // You can use a different name than "$", but you must change the refe
 	// Puts Surreal functions except for "restricted" in global scope.
 	globalsAdd() {
 		console.log(`Surreal: adding convenience globals to window`)
-		restricted = ['$', 'sugar']
-		for (const [key, value] of Object.entries(this)) {
-			if (!restricted.includes(key)) window[key] != 'undefined' ? window[key] = value : console.warn(`Surreal: "${key}()" already exists on window. Skipping to prevent overwrite.`)
-			window.document[key] = value
+		restricted = ['sugar']
+		for (const key of Object.keys(this)) {
+			const descriptor = Object.getOwnPropertyDescriptor(this, key)
+			if (!restricted.includes(key) && !key.startsWith('$')) window[key] != 'undefined' ? Object.defineProperty(window, key, descriptor) : console.warn(`Surreal: "${key}()" already exists on window. Skipping to prevent overwrite.`)
```

Comment:
I switched this to startsWith('$') because $effects was giving me a problem when switching to the descriptor mode. Could just be that I need to switch to `configurable: true` on the descriptors so they can be overwritten.